docs/install_guides/configurations-for-intel-gpu.md

```diff
@@ -11,15 +11,16 @@
 
 ## Linux
 
-If you have installed OpenVINO Runtime from the archive file, follow these steps to work with GPU:
+Once you have installed OpenVINO, follow these steps to configure it to work on GPU:
 
-1. Go to the install_dependencies directory:
+1. Download the `install_NEO_OCL_driver.sh` script from the OpenVINO GitHub repository:
    ```sh
-   cd <INSTALL_DIR>/install_dependencies/
+   curl -L https://raw.githubusercontent.com/openvinotoolkit/openvino/master/scripts/install_dependencies/install_NEO_OCL_driver.sh --output install_NEO_OCL_driver.sh
```

Comment:
why master? I suppose we need to point to 2022.2 branch

why do we need to change the way how we get this script? It's still in this `install_dependencies` folder for 2022.2

Reply:
Yes this is for 2022.2 branch. This change is suggested by BDTi, I guess it's because for pip installation, there is no <INSTALL_DIR>?

Reply:
Ok, I see this is done because of pip. In this case, we can use current approach with curl. But let's use `releases/2022/2` in URL instead of `master`